Sources/Build/Buildable.swift

```diff
@@ -9,20 +9,43 @@
 */
 
 import PackageType
+import struct Utility.Path
 
 protocol Buildable {
     var targetName: String { get }
     var isTest: Bool { get }
 }
 
+extension CModule {
+    func workingDirectory(prefix: String) -> String {
+        return Path.join(prefix, "\(c99name).build")
+    }
+}
+
 extension Module: Buildable {
     var isTest: Bool {
         return self is TestModule
     }
 
-    var Xcc: [String] {
+    func XccFlagsForPrefix(prefix: String) -> [String] {
```

Comment:
`ForPrefix` is redundant, if you want it to be explicit better to rename to `XccFlags(prefix:)`